docs/en/dce/features.md

```diff
@@ -0,0 +1,99 @@
+
+=== "[multi-cluster management](../kpanda/03ProductBrief/WhatisKPanda.md)"
+
+- Unified management of multi-cluster, supporting any Kubernetes cluster within all specific version range to be included in the container management scope, 
+
+- Rapid cluster deployment. Rapidly deploys enterprise-level Kubernetes cluster on the Web UI, quickly builds enterprise-level container cloud platform, and adapt to the underlying environment of physical machines and VMS. 
+
+- One-click cluster upgrade. Upgrade the Kubernetes version with one click, and manage the upgrade of system components. 
+
+- Cluster high avalability,the built-in cluster disaster recovery and backup abilities ensure that business system can recover from host failures, equipment room interruptions, and natural disasters, improving the stability of the production environment and reducing the risk of business interruption.
+
+- Full lifecycle management of clusters, realizing the full lifecycle management of self-build cloud native clusters. 
+
+- Open API ability, providing native Kubernetes Open API ability. 
+
+=== "Cloud native application management"
+
+- Multi-form cloud native application management, realizing the deployment and lifecycle management of different types of cloud native application, supporting cloud native application, Helm, OEM,  etc., and open multi-cloud application compatibility. 
+
+- One-stop deployment to decouple the underlying Kubernetes platform. One-stop deployment, operation and maintenance of service application to realize the lifecycle-management of application. 
+
+- Expansion and reduction of application load. It can manually/automatically expand and reduce the application load,应用负载的扩缩容，通过界面可实现应用负载的手动/自动扩缩容能力，自定义扩缩容策略，应对流量高峰。
+
+- The full lifecycle of application， supporting lifecycle-management of application such as application view, update, delete, rollback, time view, and upgrade. 
+
+- Unified load management abbility of cross-cluster. 
+
+=== "[Continuous business delivery](../amamba/01ProductBrief/WhatisAmamba.md)"
+
+- Enterprise-level cloud native CI/CD capabilities enable standardized continuous integration and delivery ability, and agile iteration of applications. It meets new business needs quickly, and improve the efficiency of application integration and release. 
+
+- Cloud-native ability of continuous delivery, containerized management of application construction environment, and custom build machine and other resources to ensure pipeline-level resource isolation, and supporting multiple types of application deployment, making the delivery of cloud-native application easier. 
+
+- Improve R&D(Research and Development) efficiency. In a visual pipeline construction environment, declarative steps are pre packaged, and complex pipelines can be created without scripts, making operation simple. The environment can be automatically created/destroyed according to the resource size to avoid resource waste. 
```

Comment:
```suggestion
- Improve R&D (Research and Development) efficiency. In a visual pipeline construction environment, declarative steps are pre-packaged, and complex pipelines can be created without scripts, making operation simpler. The environment can be automatically created/destroyed according to the resource size to avoid resource waste. 
```